src/components/Tables/DataTable/fragments/inputs/TableInputTag.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <Tag v-if="!edit" :value="data" />
+  <InputText v-else v-model="data" />
+</template>
+
+<script setup lang="ts">
+import Tag from "primevue/tag";
+import InputText from "primevue/inputnumber";
+
+export type TableInputNumberProps = { edit: boolean };
```

Comment:
```suggestion
type TableInputNumberProps = { edit: boolean };
```
se precisar usar esse tipo em outro lugar, exporte-o em `types` e importe-o aqui